api-specifications/properties.json

```
@@ -615,7 +615,7 @@
         "sizeBehavior": {
           "description": "Setting to determine how the fontsize of the label is calculated.",
           "optional": true,
-          "defaultValue": "'responsive'",
+          "defaultValue": "fixed",
```

```suggestion
          "defaultValue": "'fixed'",
```
the double quotation marks are strange, but I think they are needed